command/ca/ca.go

```diff
@@ -192,3 +194,17 @@ func completeURL(rawurl string) (string, error) {
 	// rawurl looks like ca.smallstep.com:443 or ca.smallstep.com:443/1.0/sign
 	return completeURL("https://" + rawurl)
 }
+
+// BetaCommand enables access to beta APIs.
+func BetaCommand() cli.Command {
+	return cli.Command{
+		Name:        "ca",
+		Usage:       "commads that are made available for testing new features and APIs",
+		UsageText:   "step beta ca <subcommand> [arguments] [global-flags] [subcommand-flags]",
+		Description: `**step beta ca** enables access to beta APIs..`,
```

Comment:
clarified. I don't want to mention provisioner or admin APIs since I hope that we'll add more things to beta in the future (as we test more functionality).